layouts/_partials/custom/footer.html

```diff
@@ -1,10 +1,13 @@
 <!-- Beginning of Solus Site Footer -->
 <div class="flex flex-col md:flex-row md:justify-between py-4 text-sm">
   <div class="flex flex-col md:flex-row md:gap-4 items-center pb-4 md:pb-0">
+    <p>Copyright &copy; {{ site.Params.copyright }} Solus Project.</p>
+      <div>{{ template "theme-credit" "Powered by Hextra" }}</div>
     <p>
-      Copyright &copy; {{ site.Params.copyright }} Solus Project.
+      Icons by
+      <a href="https://www.flaticon.com/free-icons/">Flaticon</a>.
     </p>
-    <div>{{ template "theme-credit" "Powered by Hextra" }}</div>
+    <p>Other brands property of their respective owners.</p>
```

Comment:
I think Zed's autoformatter doesn't work well with Hugo templates; to be fair, basically nothing does. As a result, it looks like things got a little out-of-order, and there's at least one extra, unclosed tag.